src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -36,12 +36,12 @@ const (
 
 @EmptyLine
 // UserAgent returns the UserAgent string to use when sending http.Requests.
-func @(Model.UpdateNameIfDuplicate("UserAgent", Model.PackageName))() string {
-    return fmt.Sprintf(userAgentFormat, @(Model.UpdateNameIfDuplicate("Version", Model.PackageName))(), "@(Model.PackageName)", "@(Model.ApiVersion)")
+func UserAgent() string {
```

Comment:
Is there any reason for keeping UserAgent method in the version file. Seems confusing unless it's Gopher pattern.